GliaWidgets/Component/Bubble/BubbleStyle.Accessibility.swift

```diff
@@ -0,0 +1,24 @@
+extension BubbleStyle {
+    /// Accessibility properties for BubbleStyle.
+    public struct Accessibility: Equatable {
```

Comment:
Why do they need to conform to Equatable protocol?

Reply:
I plan to add unit tests that make sure that every style that has `Accessibility` can have `Accessibility.unsupported` passed to it. So basically for XCTAssertEqual.